src/api/app/views/webui/package/_basic_info.html.haml

```diff
@@ -8,3 +8,9 @@
       %i No description set
     - else
       = render partial: 'webui/shared/collapsible_text', locals: { text: package.description }
+
+  - if User.possibly_nobody.can_modify?(package) && feature_enabled?(:responsive_ux)
```

Comment:
You need to rebase on `master`. This `responsive_ux` check was removed in #10861.

Reply:
Okay